packages/client/components/ViewerNotOnTeam.tsx

```diff
@@ -77,10 +77,10 @@ const ViewerNotOnTeam = (props: Props) => {
 export default createFragmentContainer(ViewerNotOnTeam, {
   viewer: graphql`
     fragment ViewerNotOnTeam_viewer on User {
-      meeting(meetingId: $meetingId) {
+      meeting(meetingId: $meetingId) @include(if: $meetingId) {
```

Comment:
Looks like we need a boolean after `if`

Reply:
ah that's fine, just create a boolean in the top query based off the presence of meetingId?